### PR TITLE
Add a back button for Lock screen

### LINKED ISSUE #563: Forgot Password - Go Back
screen shot 2016-08-20 at 9 34 21 am

I forgot my password and now want to go back to the opening screen so I can create a new account. Might be nice to add a way to go back:

mm_back

### PR DESCRIPTION
**Pull request by frankiebee**
fixes #563
The `<-- Back` && `Login -->` only shows if you have already created an account.
screen shot 2016-08-25 at 2 09 12 pm
screen shot 2016-08-25 at 2 09 20 pm
screen shot 2016-08-25 at 2 09 26 pm
screen shot 2016-08-25 at 2 09 35 pm

### CODE REVIEW

**Comment by flyswatter** [comment]
Great work on making this happen!

I'm happy to merge this in once you get the tests running, but I'd also like to start a discussion about the aesthetics here.

I know you are basically following the screenshot @eshon posted, and I think it worked for that one view, but for the other views, I think we could do something more consistent with the rest of the UI.

Examples:
 - On the Create/Restore view, we could add `Login` as a third button that looks the same, maybe as the first item, when a vault already exists?
 - On the restore & create screens, maybe just put `back` at the top of the view, the way we usually have it.

Also, these `back` buttons are not capitalized, the way our back buttons usually are.

Thoughts?

**Comment by frankiebee** [comment]
> On the Create/Restore view, we could add Login as a third button that looks the same, maybe as the first item, when a vault already exists?
 
i'd be willing to give it a try but i do think that it would look kind of weird 

```
LOGIN
--OR--
CREATE VAULT
--OR-- 
RESTORE VAULT
```
get rid of the ```or's```

```
LOGIN
CREATE VAULT
RESTORE VAULT
```
But then it would be very different from the initial init screen  and i kind of like the login as a small thing at the bottom corner 

>On the restore & create screens, maybe just put back at the top of the view, the way we usually have it.
If i do it for two i''d want to do it for all. To keep consistency with that button

**Comment by flyswatter** [comment]
Well we have a lot of back buttons already, and they're all at the top.

I think the login view is a decent exception, because going back from **that** screen really describes an exceptional instance, but I'm not entrenched on the point.

**Comment by frankiebee** [comment]
Okay i'll do the two at the top then. But unless you are supper set on having the login as a main button i'm going to leave it as is.

**Comment by flyswatter** [comment]
Nah I agree that one is different.

**Comment by frankiebee** [comment]
Also i noticed we dont ever say back usually we just do an arrow. Should i just do an arrow for the create and restore vault view?

**Comment by flyswatter** [comment]
Yeah, consistency is the heart of language.

**Comment by frankiebee** [comment]
screen shot 2016-08-25 at 4 04 31 pm
screen shot 2016-08-25 at 4 04 41 pm
screen shot 2016-08-25 at 4 04 47 pm
screen shot 2016-08-25 at 4 04 52 pm

**Comment by flyswatter** [comment]
One last thing…

Could we left-align that arrow?  Probably going to require an invisible element on the right to counter-balance it.

**Comment by flyswatter** [comment]
Oh nevermind, we don’t do that anywhere.  I want us to do that long term, but not in this PR.

File: ui/app/app.js [hunk 1]
```diff
@@ -299,6 +301,91 @@ App.prototype.renderDropdown = function () {
   ])
 }
 
+App.prototype.renderBackToInitButton = function () {
+  var props = this.props
+  var button = null
+  var backButton = function (style, justArrow = false) {
```
Comment (flyswatter) [inline ui/app/app.js]: Defining this method within a method looks really weird to me. I'd rather `backButton` be declared outside of this function scope, so this function is mostly about the logic used to make the render call, and not bundling it all together into one giant stream.

**Comment by flyswatter** [comment]
Added one piece of code feedback, then LGTM.

**Pull request merged**